kore.maude

```diff
@@ -220,18 +227,30 @@ Questions about this definition:
 - do we really need ~> as a K construct?  Can it be a label and thus the way K
   deal with evaluation contexts just a particular methodology and not an
   intrisic part of K ? Same question for =>.
+  Cosmin: Syntacticly at least, I would say "no" for both. From the implementation
```

Comment:
Another issue with making `~>` into a KLabel is that we'd then make no syntactic distinction between K and Item, which turned out several times to be a desired distinction.